packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -577,6 +577,27 @@ function flattenNestedBindables(
   );
 }
 
+/**
```

Comment:
If this method is just for internal usage do we really need these annotations?
I feel like the Typescript types could be explanatory enough.
But of course it also doesn't hurt to have this, besides it just taking a few more lines.